include/LightGBM/config.h

```diff
@@ -746,6 +748,10 @@ struct Config {
   // desc = separate by ``,``
   std::vector<double> label_gain;
 
+  // desc = random seed for objectives
+  // desc = used only in the ``rank_xendcg`` objective
+  int objective_seed = 1;
```

Comment:
I believe it should be `5`: `data_random_seed=1`, `feature_fraction_seed=2`, `bagging_seed=3`, `drop_seed=4`, ...

Reply:
Ah, right. Sorry about that. Fixed now.